01-arch/01-netmap.md

```diff
@@ -1 +1,55 @@
 ## Network Map
+
+Network map (or just "netmap") is a structured representation of all nodes in NeoFS network.
+Each node has multiple key-value pairs called "attributes".
+Attribute values are strings on the low level, however, some of them may be treated as integers.
+Here is an example of a simple node:
+```
+Continent=Europe
+Country=Germany
+Price=1000
+Capacity=1000
+```
+
+Some node attributes can be grouped (e.g. (geographical ones)[]), thus creating
+a graph representation for the network map.
+Strictly speaking it is a forest of rooted trees where leaves (single nodes) are shared.
+Every tree represent a single attribute group.
+For example, geographical attributes can be naturally ordered:
+`Continent`, `Country`, `City`, `DC` (data center) or any other place identifier.
+
+### Epoch
+For multiple nodes to have consistent view of network netmap should be immutable.
+It changes in regular time periods called epochs.
+During an epoch netmap is immutable however new nodes can be registered and bad (offline) ones can be removed. 
+All these changes signed by Inner Ring are propagated to the network when new epoch starts.
+
+### Predefined attributes
+Some of the attributes are reserved and follow specific validation rules,
+such as (`UN-LOCODE`)[https://en.wikipedia.org/wiki/UN/LOCODE] should be well defined.
```

Comment:
It isn't clear what the UN-LOCODE is - is it an attribute or a set of validation rules?